Modules/posixmodule.c

```diff
@@ -1860,6 +1860,7 @@ static PyStructSequence_Field statvfs_result_fields[] = {
     {"f_favail", },
     {"f_flag",   },
     {"f_namemax",},
+    {"f_fsid", },
```

Comment:
Really minor nit: closing ``}`` should be aligned with those above for local consistency.